README.md

```diff
@@ -70,6 +70,42 @@ the first start can fail if downloading the images takes more than the default s
 
 The containers should start on the next boot automatically.
 
+# Alternative single-pod deployment
+
+## SELinux
+On SELinux-enabled systems, the context of mapped host directories needs to be set manually. If all the mapped directories are under `/path/to/immich`, set the context with
+```
+chcon -R -t container_file_t /path/to/immich
+```
```

Comment:
i think that's applicable to all deployments. Adding the `z` (or `Z` i never remembers which) flag on volume mounts automatically relabels the host directories.

Reply:
I'm nor clear what you mean here, you want to remove the SElinux mention altogether?

At least with the Podman version in Alma 9, SELinux context is set automatically when creating single containers (with `:z` or `:Z`), but not when creating a pod with `kube play`.

Podman kube play supposedly supports setting the context for volumes with annotations, but that hasn't worked in my lab so far. Setting it manually is simpler to explain.

I don't know if there's a way to set the context in Quadlet.

`Z` sets the context so all containers can access the mount, while `z` sets it so only the one specific container can access it. The `chcon` command here would be equivalent to `-v /path:/mnt:Z`.

Reply:
Thanks for the detailed explanation. I think it's good to keep the note, maybe at the end of the readme in a "misc" section? (I could also add details about GPU passthrough for the machine learning container there as well)